### DISCUSSION TITLE
I can not install Foocus

### DISCUSSION THREAD

**Discussion by AndreiIvanescu** [lllyasviel/Fooocus#3626] [Q&A]
![image_2024-09-18_201359709](https://github.com/user-attachments/assets/2206a228-3fa9-4d8d-9f48-74afff93a748)
It's my 12th attempt this day to install focus. For whatever reason, the installation goes to 100% and then nothing happens, foocus doesn't open and I can't do anything. I tried everything, from downloading juggernaut from their website and replacing it in the folder to using Pinokio, where I got the same error. Does anyone has any clue what can be wrong? Also I had Foocus installed before on this PC, but I deleted it.

**Comment by kalle07** [comment]
https://github.com/lllyasviel/Fooocus/discussions/3616

mostly stucks if low space on SSD or somewhat block in firewall ...
Unfortunately i dont know which models foocus needs to start
